src/endpoints/Stories/types.ts

```diff
@@ -104,9 +113,76 @@ export interface SlateStoryUpdateRequest extends GenericUpdateRequest {
     /**
      * Intro field is not supported for Prezly Content Format stories.
      */
-    intro: never;
-    content: PrezlyContentFormat;
+    intro?: never;
+    content?: PrezlyContentFormat;
+    autosaved_content?: PrezlyContentFormat;
+    content_version?: number;
 }
 
 export type CreateRequest = HtmlStoryCreateRequest | SlateStoryCreateRequest;
 export type UpdateRequest = HtmlStoryUpdateRequest | SlateStoryUpdateRequest;
+
+export interface AutosaveRequest {
+    autosaved_content: PrezlyContentFormat;
+    content_version?: number;
+}
+
+export interface RevertRequest {
+    content_version?: number;
+}
+
+export interface PublishRequest {
+    published_at?: Iso8601DateTime;
+    visibility?: Story.Visibility.PUBLIC | Story.Visibility.PRIVATE | Story.Visibility.CONFIDENTIAL;
+}
+
+export interface UnpublishRequest {
+    visibility?: Story.Visibility.PUBLIC | Story.Visibility.PRIVATE | Story.Visibility.CONFIDENTIAL;
+}
+
+export interface ScheduleRequest {
+    publish_at?: Iso8601DateTime;
+    visibility?: Story.Visibility.PUBLIC | Story.Visibility.EMBARGO;
+}
+
+export type UnscheduleRequest = UnpublishRequest;
+
+const EXTENDED_STORY_INCLUDED_ONDEMAND_FIELDS_SHAPE /*: Record<
+    keyof Omit<ExtendedStory, keyof Story>,
+    boolean
+>*/ = {
```

Comment:
Why couldn't you use the type here? Even if `satisfies` operator is not yet ready, you can still use the checks from implicit typing.

Reply:
:+1: Indeed. I've been trying this, but for some reason, it didn't work before. Tried again, and it's good. :man_shrugging: 

Will change. Thanks!